extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -492,7 +491,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		let notebookConfig = vscode.workspace.getConfiguration(constants.notebookConfigKey);
 		let maxDepth = notebookConfig[constants.maxBookSearchDepth];
 		// Use default value if user enters an invalid value
-		if (isNullOrUndefined(maxDepth) || maxDepth < 0) {
+		if (maxDepth === null || maxDepth === undefined || maxDepth < 0) {
```

Comment:
Can you provide context on this change?

Reply:
I got a warning from IntelliSense saying that the method is deprecated since v4.0.0